src/app/components/LoginForm/index.tsx

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import React from 'react';
+import i18next from 'i18next';
+import { useForm } from 'react-hook-form';
+
+import Input from '~components/Input';
+import { emailRegex } from '~utils/inputValidations';
+import { AUTH_FIELDS } from '~constants/fields';
+import { ContentForm } from '~utils/types';
+import { LoginBody } from '~screens/Login/types';
+
+import styles from './styles.module.scss';
+
+function LoginForm({ onSubmit, isLoading }: ContentForm) {
+  const { register, handleSubmit, errors, formState } = useForm<LoginBody>({
+    mode: 'all'
+  });
+
+  return (
+    <>
```

Comment:
fragment not necessary

Reply:
You need a container for the form and a div, if it is removed it complains

Reply:
sorry didn't see the div below